spec/bolt/cli_spec.rb

```diff
@@ -2100,6 +2100,12 @@ def stub_config(file_content = {})
       cli.execute(cli.parse)
     end
 
+    it 'lists targets with resolved configuration' do
+      cli = Bolt::CLI.new(%w[inventory show -t localhost --detail])
```

Comment:
Can we also verify that the output of this is what we expect? Last comment and then I think this is GtG!